core/templates/base-components/header.template.html

```diff
@@ -10,7 +10,7 @@
 <meta property="og:site_name" content="Oppia">
 <meta property="og:description" content="<%= htmlWebpackPlugin.options.meta.description %>">
 
-<link rel="apple-touch-icon" href="/assets/images/logo/favicon.png">
+<link rel="apple-touch-icon" href="/assets/images/logo/favicon.webp">
```

Comment:
Does Apple support WebP favicons?

Reply:
for this issue i have added another link tag for png